packages/clerk-js/src/core/clerk.ts

```diff
@@ -2018,7 +2022,7 @@ export class Clerk implements ClerkInterface {
   };
 
   // TODO: Be more conservative about touches. Throttle, don't touch when only one user, etc
-  #touchLastActiveSession = async (session?: ActiveSessionResource | null): Promise<void> => {
+  #touchCurrentSession = async (session?: AuthenticatedSessionResource | null): Promise<void> => {
```

Comment:
Renaming away from "last active", as it resembles a lot from the `active` status, and we're trying to `/touch` against the last current/selected session on the client here